pkg/cluster/spec.go

```diff
@@ -589,6 +589,9 @@ func (vb *VersionsBundle) Manifests() map[string][]v1alpha1.Manifest {
 	// Cilium manifest
 	manifests["cilium"] = []v1alpha1.Manifest{vb.Cilium.Manifest}
 
+	// Kindnetd manifest
+	manifests["kind"] = []v1alpha1.Manifest{vb.Kindnetd.Manifest}
```

Comment:
should this be kind or kindnetd?

Reply:
Since the "bundle" is called `Kindnetd` I would go with kindnetd